de-lib/src/main/java/org/iplantc/de/client/models/tool/sharing/ToolSharingRequest.java

```diff
@@ -0,0 +1,21 @@
+package org.iplantc.de.client.models.tool.sharing;
+
+import com.google.web.bindery.autobean.shared.AutoBean;
+
+import java.util.List;
+
+/**
+ * Created by sriram.
+ */
+public interface ToolSharingRequest {
+
+    String getUser();
```

Comment:
You probably updated this later, but I believe this was updated to have a `SharingSubject` instead of `user`